src/v/bytes/random.h

```diff
@@ -22,4 +24,6 @@ bytes get_crypto_bytes(size_t n = 128 * 1024, bool use_private_rng = false);
 // Makes an random alphanumeric string, encoded in an iobuf.
 iobuf make_iobuf(size_t n = 128);
 
+iobuf split_as_iobuf(std::string_view str, int n_fragments);
```

Comment:
I think this should go into a different header

Reply:
any preference where about?
or maybe would it look better here if it accepted `bytes` instead of `std::string_view`?

Reply:
It is, as it chooses fragment sizes randomly.